bin/infrasim-main

```diff
@@ -35,32 +34,35 @@ if __name__ == '__main__':
               "Please check infrasim configure file: {}".format(INFRASIM_CONF)
         sys.exit(-1)
 
+    node = model.CNode(conf)
+
     try:
         if len(sys.argv) < 2:
             print "{} start|stop|status|restart|version".format(sys.argv[0])
             sys.exit(0)
 
         if sys.argv[1] == "start":
-            socat.start_socat()
-            ipmi.start_ipmi()
+            node.init()
+            node.precheck()
+            node.start()
             print "Infrasim service started.\n" \
-              "You can access virtual {} via vnc:{}:5901".format(node, netifaces.ifaddresses(eth)[netifaces.AF_INET][0]['addr'])
+              "You can access node {} via vnc:{}:5901".\
+                format(node.get_node_name(),
+                       netifaces.ifaddresses(eth)[netifaces.AF_INET][0]['addr'])
         elif sys.argv[1] == "stop":
-            qemu.stop_qemu()
-            ipmi.stop_ipmi()
-            socat.stop_socat()
+            node.init()
```

Comment:
A "init()" is to read configuration file and let node object know the necessary information.
A "init()" is NOT going to start anything.

Let the node know key information, including the target infrasim name and workspace, then stop it.